<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordWidget</class>
 <widget class="QWidget" name="WordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_addr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>地址设定</string>
   </property>
   <widget class="QLineEdit" name="edit_operatingAddr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>21</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>字地址： </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>数据类型：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dataType">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>16位整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16位正整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位正整数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16位BCD码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位BCD码</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32位浮点数</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_wordAddr">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>20</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_touch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>操作设定</string>
   </property>
   <widget class="QComboBox" name="comboBox_operateType">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>21</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>写入数据</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>数值加</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>数值减</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>连续加</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>连续减</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>操作类型：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>45</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>常数：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_value">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>45</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_cycle">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>23</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>绕回</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>181</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>最小值： </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_min">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>最大值： </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_max">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_wordAddr_const">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>76</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_const">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>19</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>动态控制</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_const">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>78</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>77</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
